src/onLoad.js

```diff
@@ -0,0 +1,55 @@
+import $ from "jquery";
+
+export default (editor, opts = {}) => {
+
+    $(function() {
+        let css =`.column-actions
+        {
+            background-image: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="none"><path stroke="%23ffffff" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M3 3h3M3 21h3m0 0h4a2 2 0 0 0 2-2V9M6 21V9m0-6h4a2 2 0 0 1 2 2v4M6 3v6M3 9h3m0 0h6m-9 6h9m3-3h3m0 0h3m-3 0v3m0-3V9"/></svg>');
```

Comment:
Would it not be better to just store the images as .svg files?

Reply:
Maybe, tbh I'm not sure how to do that within an npm package so went with what I do know.

Reply:
Honestly didn't really think of that at the moment. Lets just keep it as it is.